src/include/hardware.h

```diff
@@ -73,6 +73,23 @@
 #define WT_WRITE_ONCE(v, val) WT_RELEASE_WRITE_WITH_BARRIER(v, val)
 #endif
 
+#ifdef HAVE_NEOVERSE
+#define WT_ACQUIRE_READ(v, val)                                         \
+    do {                                                                \
+        if (sizeof((val)) == 1) {                                       \
```

Comment:
Is the branch switch more costly than any performance gain we could get?

Reply:
There is no branch switch, per my description the branches are compiled out.